src/components/Button.jsx

```diff
@@ -1,13 +1,22 @@
+import PropTypes from 'prop-types';
+
 const Button = (props) => {
-  const { id, title } = props;
   return (
-    <>
-      <button className="bg-blue-400 hover:bg-blue-500 active:bg-red-100 transition p-3 rounded-3xl w-full">
-        <span className="mr-1">{id}.</span>
-        {title}
-      </button>
-    </>
+    <button
```

Comment:
please use Button component

Reply:
it's Button component. I deleted "<></>" tag because I have only button

Reply:
it is tag <button> should be <Button/> component